src/lib/action.rs

```diff
@@ -35,29 +35,37 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-// #![warn(missing_docs)]
+#![warn(missing_docs)]
```

Comment:
I'm not convinced this is a great setting to leave on: sometimes it just means we end up adding completely pointless documentation.